scripts/performance/install-perfline.sh

```diff
@@ -69,7 +69,29 @@ function update_inventory_host()
 function install_perfline()
 {
     local host_file="inventories/perfline_hosts/hosts"
-    rm -rf seagate-tools
+
+    if [ -f "$task_status" ]
```

Comment:
try to use a list of files/folder and then loop over list to check and delete it.